### PR TITLE
nativefier: add German translation

### PR DESCRIPTION
**Pull request by ProfileID**
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

Added German translation to nativefier.

### CODE REVIEW

File: pages.de/common/nativefier.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# nativefier
+
+> Befehlszeilen-Tool zum Erstellen einer Desktop-Anwendung für jede Website mit minimaler Konfiguration.
+> Weitere Informationen <https://github.com/jiahaog/nativefier>.
```
Comment (navarroaxel) [inline pages.de/common/nativefier.md:4]: The `exa` page use `Mehr`. And you missed the `:`
```suggestion
> Mehr Informationen: <https://github.com/jiahaog/nativefier>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @ProfileID, thanks for your submission. I left a comment.

File: pages.de/common/nativefier.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Erstellen einer Desktop-Anwendung für eine Website:
+
+`nativefier {{URL}}}`
+
+- Erstellen Sie eine Desktop-Anwendung mit einem benutzerdefinierten Namen:
+
+`nativefier --Name {{Name}} {{URL}}`
```
Comment (sbrl) [inline pages.de/common/nativefier.md:12]: I think we have a casing issue with the argument here. The original page uses `--name`, but here we have `--Name`?

**Review by sbrl — COMMENTED** [review]
Thanks for the new page, @ProfileID! I've left a comment below for you to review.

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]
Do not know the language but seems good to me.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks, @ProfileID  :slightly_smiling_face:

**Pull request merged**